examples/aws-lambda/README.md

```diff
@@ -0,0 +1,27 @@
+# Honeybadger AWS Lambda - Serverless Example
```

Comment:
Here are a few example projects that we may need to update after this is done:

https://github.com/honeybadger-io/serverless-quickstart
https://github.com/honeybadger-io/honeybadger-lambda-node

I think it would make sense to update serverless-quickstart and deprecate/archive the honeybadger-lambda-node repo (which seems similar to this example?)

Reply:
We may also want to link to this example project in the serverless-quickstart readme.

Reply:
I'd also consider archiving both of those repos if they are both redundant. :)

Reply:
>https://github.com/honeybadger-io/honeybadger-lambda-node

I think this is redundant now that we have the aws-lambda example project.

>https://github.com/honeybadger-io/serverless-quickstart

This appears outdated. If it's still being used from the team as a boilerplate, it makes sense to update (and maybe link to the aws-lambda example project), otherwise it could also be archived. `serverless.com` provides a dashboard and a cli so you don't have to log into AWS Console to configure env variables.
However, I should note that our serverless integration only supports AWS Lambda. In the future, we could extract the aws-lambda implementation to a separate _serverless_ package/repository and add support for more cloud providers (azure, google cloud).

Reply:
I think I'll archive both of these projects after this PR is finished. I can update their readmes to point to the new example directory in this repo.